### PR TITLE
add comments

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: adhocracy4/comments/services.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from .models import Comment
+
+
+def delete_comments(contenttype, pk):
+    comments = Comment.objects.all().filter(
+        content_type=contenttype, object_pk=pk)
+    for comment in comments:
+        comment.delete()
```
Comment (xi) [inline adhocracy4/comments/services.py]: What is this used for?

Comment (vellip) [inline adhocracy4/comments/services.py]: @MagdaN you know that?

Comment (MagdaN) [inline adhocracy4/comments/services.py]: @xi the code gets all comments for a resource and deletes them, e.g. it is used when an idea is deleted it is used to delete all comments of that idea

Comment (xi) [inline adhocracy4/comments/services.py]: Ok. But where is it called? More specifically: In which cases is this used and in which cases is the code in adhocracy4/comments/signals.py used?

Comment (2e2a) [inline adhocracy4/comments/services.py]: signaly.py registers a delete handler for the post_delete signal.

However, @slomo has created a generic function for this:
`generics.setup_delete_signals(settings.A4_COMMENTABLES, Comment)`

I'll create a pull request.

**Pull request merged**